tests/e2e/actor_reminder/actor_reminder_test.go

```diff
@@ -504,9 +504,9 @@ func TestActorReminderTTL(t *testing.T) {
 	// Set reminder
 	reminder := actorReminder{
 		Data:    "reminderdata",
-		DueTime: "1s",
-		Period:  "PT2S",
-		TTL:     "9s",
+		DueTime: "10s",
+		Period:  "PT5S",
```

Comment:
Do we not want to keep this a smaller number to have a larger sample size?

Reply:
I am working with the assumption that some runs might get dropped if too granular.